File: leetcode/algorithms/string/valid-anagram.ts
function getAnagramSig(s: string): string {
    return s.split('').sort((a: string,b: string): number => a.charCodeAt(0) - b.charCodeAt(0)).join('');
}
function isAnagram(s: string, t: string): boolean {
    return getAnagramSig(s)===getAnagramSig(t);
};

File: leetcode/algorithms/string/valid-anagram.java
import java.util.Arrays;

class Solution {
    public String getAnagramSig(String s){
        char[] letters = s.toCharArray();
        Arrays.sort(letters);
        return new String(letters);
    }
    public boolean isAnagram(String s, String t) {
        return getAnagramSig(s).equals(getAnagramSig(t));
    }
}